lib/vmdb/console_methods.rb

```diff
@@ -24,6 +24,20 @@ def backtrace(include_external = false)
       caller.select { |path| include_external || path.start_with?(Rails.root.to_s) }
     end
 
+    def set_translation_locale(locale = "en")
+      I18n.locale = locale
+    end
+
+    def set_user_translation_locale(userid: "admin", locale: "en")
+      user = User.find_by_userid(userid)
+
+      user.settings[:display] = {:locale => locale}
+      user.save!
```

Comment:
Curious if setting the translation locale here could be optional, allowing this method to just load the current translation setting if no args are given, instead of just setting it to `"en"` by default.

I could see this method being called when the console first starts to allow translations to be configured by default when the console is loaded.